Kommunicate-io/Kommunicate-Web-SDK
# FW-1953 Pre-Chat lead form - Dropdown list on Web

### What do you want to achieve?

- Create a dropdown list for a pre-chat lead form field.


### PR Checklist


### How was the code tested?


- Load installation script with preLeadCollection as suggested in the below image.

- Check a dropdown for that field will appear in the chat widget.
---
![image](https://user-images.githubusercontent.com/22856546/111276173-11952680-865d-11eb-8794-437642d3aa5a.png)
### What new thing you came across while writing this code? 
-

### In case you fixed a bug then please describe the root cause of it? 
-

NOTE: Make sure you're comparing your branch with the correct base branch
